src/plugins/charts/BarGraphView.vue

```diff
@@ -84,23 +84,31 @@ export default {
             const metadata = this.openmct.telemetry.getMetadata(telemetryObject);
             this.telemetryObjectFormats[key] = this.openmct.telemetry.getFormatMap(metadata);
             const telemetryName = telemetryObject.name;
+            const telemetryType = telemetryObject.type;
+            const telemetryObjectPath = [telemetryObject, ...this.path];
+            const telemetryIsAlias = this.openmct.objects.isObjectPathToALink(telemetryObject, telemetryObjectPath);
 
-            // if the existing telemetry name is absent, or different, mutate the object
-            let existingTelemetryName;
+            this.addAndMutateStyleConfig(key, 'name', telemetryName);
```

Comment:
Let's try to reduce these 3 calls into 1

Reply:
going to try something like this:
`this.openmct.objects.mutate(
                    this.domainObject,
                    `configuration.barStyles.series[${key}]`,
                    seriesProperties
                );`

Reply:
Ok, changed to a single update, though had to do changes to account for an existing styles object.